data/donaina.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "Aina Ayoola",
+	"displayStatsPublic": "true",
```

Comment:
hi, @donaina after the recent update "displayStatsPublic": "true",` is no longer needed. You can delete this line. 
``` suggestion
	
```